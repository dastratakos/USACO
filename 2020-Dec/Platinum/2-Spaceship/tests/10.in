60 56 60
000111110110101110111001101000110110110010011101111001111001
010111110111110101011101111111110011010111101111011111001111
101011111011101110101011111100010011010111101101101011101100
101110110100111011110110111111111111110101111111011111110111
000111100010111001101110111010001001110101010101001110111100
011101101101011100110000100111111010100111011011111110111011
010111100111011111110010111111010111000111111110110110110011
001101111001010011101111111001011111111111001010111110100111
101111111111011111001111010010111011111111111001100111111101
001011010101110010101111111101101111110111111111011011011011
111111101011111101011111010011011111101000110111101111101010
111101101110111011111101111101100101111101111000000111100101
111111111011110111101110111011001111111111101110010111010010
011011101100011110111011011011111110101010011011110101111010
011001010111010110011001011111100011011111110011101111111101
111111010110001111111000010011011110101101110010011011100001
011000110111101101110011001011101111001011111010101111011011
111001001000011111101101101100001100001011001101010111111100
110011101111110110111111001101010010110010111011010111011101
101111010110001011110111010001011110101111111111000111000001
011110010010010110111111110110111111100010001010101111111100
010111111111110101010110111010111011111011101111000001111111
011111011100011011101011111110101010111011111100010100111111
000001010100100111111111001110101101111010111001111111111111
011010100010011111011101110111110111111110111011111111111011
101010111111111101110110100101101011100110111011111110101011
110011100011001110110100111111011011101010011111011111011111
111011111010000110111111100011111101010101101101010110010101
111100010111101011101100011111001000101111011011111111010111
111010011011011011011111011111011111101011101100101111111011
011000100110101111111011011011111101111110011011110111101011
101100101110011111101111110001000010111011111111111111111101
111000111000110101111010100011001111011111000100001111111111
010111111000111101111001101111111010110110010010111111001001
101111101101011111011111111111101001110100100111011101011110
000010101101110011011001110011001111000111101111000110111010
011000110110000111110010100000101111110001101011010111110110
001111010111011111101111100000101111111100101011001100011111
111111111101111111111111110100110010110111111011110100010110
011001101101111010100111011101111001111010011111111100001110
011110101011001100101111111001101010110101111110110101011010
111011001101011111011111111111110101111010011101111011010101
011111100001101111011110001101010111111111101111110110111001
001011111100011111111111011110111001011101011001001011110010
111000001010101111011011001111011110101110110111111011101101
001111010010111001110111010110010111110111101101111110101111
110011100010110011111100101111011100111101111110110110110110
011001001011101111110111001111011011011010011100100111011111
111101111111110011011100101110111110111111111111101011010110
011011000011111011111001111001110100001100101101101001110010
110111110111001110101111111110100111011111111010100001010000
110110111111011011111111111001100000111111101111011111101001
011101011101010011011001011111110110111100010011101111101001
010010111111111111101110001110011110011011111011111101000111
011111111010111011110101011111110011000101110110010101111111
110010111101101101111011101011111111110111111000010111101011
101100101011110010100110110011010001111010000011111101110010
010011111011110111001100110101111011011111110011010111111111
011111110110100011100110011101111001111101011000111010100010
010110010100111110100011111111011111111111101101000101110111
55 18 56 34
55 50 56 29
55 27 56 48
55 14 56 43
55 46 56 43
55 18 56 27
55 22 56 29
55 22 56 34
55 51 56 45
55 28 56 22
55 8 56 14
55 35 56 1
55 21 56 7
55 16 56 20
55 52 56 33
55 36 56 52
55 33 56 18
55 42 56 36
55 56 56 59
55 14 56 3
55 3 56 10
55 47 56 36
55 5 56 17
55 54 56 41
55 49 56 33
55 5 56 40
55 29 56 40
55 8 56 51
55 1 56 8
55 8 56 2
55 37 56 41
55 31 56 29
55 60 56 29
55 11 56 52
55 21 56 30
55 54 56 57
55 7 56 43
55 36 56 38
55 51 56 12
55 10 56 22
55 51 56 38
55 11 56 6
55 13 56 26
55 14 56 52
55 19 56 50
55 4 56 15
55 14 56 53
55 44 56 26
55 35 56 29
55 22 56 26
55 36 56 46
55 22 56 40
55 5 56 44
55 27 56 32
55 49 56 51
55 38 56 23
55 20 56 52
55 18 56 49
55 28 56 43
55 10 56 48
