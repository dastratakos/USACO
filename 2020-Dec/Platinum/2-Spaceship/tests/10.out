768536213
124663212
335536513
9110456
904182673
952262475
831382696
225685797
516113573
688308301
308174105
26942683
719419822
535665185
301255547
674135617
587564459
966634983
980309487
842497672
353693723
750282725
267963114
497874423
562850263
963506235
569097989
410402711
317934104
959620032
642599368
552356199
854108596
873318608
819812349
571358371
895075914
275542421
12750515
163330953
626560137
576697751
600673812
83365043
989007431
350958011
739545664
430565036
691093365
851430370
520182992
146798621
714859603
29968806
46439401
256898941
51278276
482698333
965548457
275623426
